ogr/ogrsf_frmts/gml/ogrgmldriver.cpp

```diff
@@ -196,6 +196,12 @@ void RegisterOGRGML()
         "    <Value>YES</Value>"
         "    <Value>NO</Value>"
         "  </Option>"
+        "  <Option name='OGR_SCHEMA' type='string' description='"
+        "   Override one or more auto-detected field definitions by providing "
+        "a JSON list "
+        "of field definitions."
+        "   This can be a filename or a JSON string."
+        "   or a URL.  If not provided, no validation is done.'/>"
```

Comment:
What does "If not provided, no validation is done" mean?

Reply:
it's nonsense: removed. I will also update the text with the rst doc.

I am open for suggestion for better wording.